src/executables/util_service.c

```diff
@@ -1668,16 +1669,16 @@ process_server (int command_type, int argc, char **argv, bool show_usage, bool c
 	      print_message (stdout, MSGCAT_UTIL_GENERIC_START_STOP_3S, PRINT_SERVER_NAME, PRINT_CMD_START, token);
 
 #if !defined(WINDOWS)
-	      if (check_ha_mode == true)
+	      status = sysprm_load_and_init (token, NULL, SYSPRM_IGNORE_INTL_PARAMS);
+	      if (status != NO_ERROR)
 		{
-		  status = sysprm_load_and_init (token, NULL, SYSPRM_IGNORE_INTL_PARAMS);
-		  if (status != NO_ERROR)
-		    {
-		      util_log_write_errid (MSGCAT_UTIL_GENERIC_SERVICE_PROPERTY_FAIL);
-		      print_result (PRINT_SERVER_NAME, status, command_type);
-		      break;
-		    }
+		  util_log_write_errid (MSGCAT_UTIL_GENERIC_SERVICE_PROPERTY_FAIL);
+		  print_result (PRINT_SERVER_NAME, status, command_type);
+		  break;
+		}
```

Comment:
Move `#if !defined(WINDOWS)` after `sysprm_load_and_init()`. The single_node configuration must be known on windows too.

Reply:
moved it now